source/get-started/containers/container-images.rst

```diff
@@ -0,0 +1,105 @@
+.. _container-images:
+
+|CL-ATTR| container images
+##########################
+
+|CL| can run inside of a container on top of any operating system as long as
```

Comment:
[Simplify intro to two paras. ]

|CL| runs inside of a container on top of any operating system that hosts a containerized environment such as Docker\* or Kubernetes\*. A |CL| base image is available, as well as popular application images built from the |CL| base image. 

Browse all |CL| container images on `the Docker Hub* website <https://hub.docker.com/search?q=clearlinux&type=image>`_.  Find the Dockerfiles used to build |CL| container images `on GitHub
<https://github.com/clearlinux/dockerfiles>`_.

Reply:
This is much better. Thanks!